packages/manager/apps/dedicated/client/app/css/less/common/table.less

```diff
@@ -63,3 +63,26 @@ table {
 .table-striped > tbody > tr:nth-child(2n) > th {
   &:extend(.oui-background-p-050);
 }
+
+.list-striped,
+.list-striped .oui-collapsible__body {
+  padding: 0 !important;
+}
+
+.list-striped oui-radio {
+  margin-top: 0 !important;
+  padding: calc(1rem - 1px);
+}
+
+.list-striped oui-radio:nth-child(2n) {
+  &:extend(.oui-background-p-050);
+}
+
+.separator {
+  margin: 0 0.5rem;
+}
+
+.flright {
+  margin-left: auto;
+  margin-right: 0;
+}
```

Comment:
**nitpick:** Use `ml-0` and `mr-auto` instead of cutom CSS.

Reply:
I don't remember why I have added the flright class :)... I think I can remove it